### PR TITLE
Data 3149

### PR DESCRIPTION
**Pull request by evilpixie87**
Files modified for latest errata, Additional Resources, and Campaign Clarifications

### CODE REVIEW

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_equip_general.lst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# CVS $Revision: $ $Author: $ -- Fri Jun 17 21:51:56 2016 -- reformated by PCGen PrettyLST v6.06.00
+# CVS $Revision: $ $Author: $ -- Fri Jan 29 20:50:52 2016 -- reformated by PCGen PrettyLST v6.06.00
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_equip_general.lst]: This means you're working with majorly out of date files... I should never see dates rolling backwards.

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities_class.lst [hunk 2]
```diff
@@ -43,7 +43,7 @@ Rage Powers				KEY:Viking ~ Rage Powers			CATEGORY:Special Ability	TYPE:FighterC
...
+Defend Charge			KEY:Witchguard ~ Defend Charge		CATEGORY:Special Ability	TYPE:ClassFeatures.RangerClassFeatures.SpecialQuality.Extraordinary.RangerHuntersBond																																																																																																																													DESC:A witchguard forms a bond with a spellcaster he has sworn to defend. This bond allows the witchguard to spend a move action to grant an adjacent spellcaster a +%1 dodge bonus to AC and a +%1 circumstance bonus on concentration checks. The bonuses last for %2 rounds. The witchguard can use this ability %3 times per day.|(RangerLVL/4)*2|max(1,WIS)|floor((RangerLVL-1)/3)																																																																																																																																																																																																																SOURCEPAGE:p.25
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities_class.lst]: Why?

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities.lst [hunk 3]
```diff
@@ -7,33 +7,46 @@ SOURCELONG:People of the North	SOURCESHORT:POTN	SOURCEWEB:http://paizo.com/produ
 # New Traits
```
Comment (LegacyKing) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities.lst]: What actually changed here? Looks like a file rewrite...

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities_class.lst (hunk 2 above)
Comment (evilpixie87) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities_class.lst]: http://paizo.com/pathfinderSociety/rpg/clarifications#RPG

People of the North
Page 25—Replace the third sentence of the Witchguard's Defend Charge ability with the following. "At 8th level and every 4 levels thereafter, these bonuses increase by 2." Replace the second-to-last sentence of the ability with the following. "At 7th level and every 3 levels thereafter, the witchguard can use this ability one additional time per day."

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_equip_general.lst (hunk 1 above)
Comment (evilpixie87) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_equip_general.lst]: I didn't actually mean to include this file. It was part of an early clean-up I did to better see what fixes were needed to implement PFS. Sorry. I'll pull this!

File: data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities.lst (hunk 3 above)
Comment (evilpixie87) [inline data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities.lst]: I made those traits that required it ethnicity-specific by adding PRETEMPLATE to them, and I added the snowcaster ethnicity "ability" as well as some human ethnicity "abilities" so they can get the proper templates.

**Comment by LegacyKing** [comment]
@evilpixie87 - Please look into the recommendations from the PFS maintainer:

I looked over this PR.. PFS support should be implemented exclusively within the _pfs folder under its parent source.

Add EXCLUDEs to the _pfs_.pcc file for those parent sources that should be completely excluded in PFS.

For items that need to have a different behaviour, add a _pfs*.lst file that includes the .MODs, .COPYs, etc that need to be applied to the original source in order to make it PFS complaint. Then add its reference in the _pfs_.pcc file with a PRECAMPAIGN tag, like below. This PRECAMPAIGN is needed because the entire pfs_*.pcc file will be scanned whether or not the parent source is actually loaded and we need to make sure the PFS specific changes are only loaded if the parent source was loaded.
ABILITY:pfs{file}.lst|PRECAMPAIGN:1,INCLUDES={original source's key}

The only change to the parent source will be to add |Pathfinder_PFS to it's GAMEMODE. ... THERE SHOULD BE NO OTHER CHANGE TO THE PARENT SOURCE UNLESS 1) There is no other way to make it PFS compatible, AND 2) You are absolutely sure that the change will not affect any -PFS character now or in the future.

Lastly, you need to add the new source in data\pathfinder_pfs\paizo\pathfinder_society\roleplaying_guild_guide_roleplaying_guild_guide.pcc.

The above should probably be added to the PFS docs for PCGen.

**Pull request merged**